Task_1/Feeder.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    class Feeder
+    {
+        Random rand = new Random();
```

Comment:
Each member should have access modifier.
Initialization should be performed in constructor.